src/foundation.cs

```diff
@@ -8194,16 +8194,16 @@ interface NSInputStream {
 
 	}
 
-	delegate bool EnumerateLinguisticTagsEnumerator (NSString tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
+	delegate bool NSEnumerateLinguisticTagsEnumerator (NSLinguisticTagScheme tag, NSRange tokenRange, NSRange sentenceRange, ref bool stop);
 
 	[Category]
 	[BaseType (typeof(NSString))]
 	interface NSLinguisticAnalysis {
 		[Export ("linguisticTagsInRange:scheme:options:orthography:tokenRanges:")]
-		string[] GetLinguisticTags (NSRange range, string scheme, NSLinguisticTaggerOptions options, [NullAllowed] NSOrthography orthography, [NullAllowed] out NSValue[] tokenRanges);
+		string[] GetLinguisticTags (NSRange range, NSLinguisticTagScheme scheme, NSLinguisticTaggerOptions options, [NullAllowed] NSOrthography orthography, [NullAllowed] out NSValue[] tokenRanges);
```

Comment:
you need to bind `NSString` and add `[Advanced]` to the API (if the constants can be extended), otherwise it can be internal
then you can use a `[Wrap]` signature that converts from the smart enum into an `NSString`